astroplan/data/observatories.json

```diff
@@ -381,4 +381,18 @@
         "name": "mro",
         "source": "MRO website"
     }
-}
\ No newline at end of file
+    "SALT": {
+        "source" : "SALT website",
+        "elevation": 1700, 
+        "name": "South African Large Telescope", 
+        "longitude_unit": "degree", 
+        "latitude_unit": "degree", 
+        "latitude": 20.810499, 
+        "elevation_unit": "meter", 
+        "longitude": 32.561633, 
```

Comment:
[Wikipedia](https://en.wikipedia.org/wiki/Southern_African_Large_Telescope) says that the position is 32°22′33.62″S 20°48′38.44″ E – did you swap the latitude and longitude? Also, eastern longitudes are negative.